### ISSUE TITLE
To Love-Ru Trouble Darkness: True Princess [PCSG00689]

### ISSUE THREAD

**Issue by Zangetsu38** [Vita3K/compatibility#245]
# Game summary
- Game name: To Love-Ru Trouble Darkness: True Princess
- Game serial: PCSG00689
- Game version: 1.00

# Vita3K summary
- Version: v0.1
- Build number: 1076
- Commit hash: https://github.com/vita3k/vita3k/commit/bc03af6

# Test environment summary
- Tested by: Zangetsu38
- OS: Windows 10
- CPU: AMD Ryzen 2700X
- GPU: NVIDIA GTX 1080 Ti
- RAM: 16 GB DDR4 3600 mhz

# Issues
- stop boot on mono-vita with unicorn error
Unicorn error 0x7 at: start PC: 0x82E023D0 error PC 0x83F83FD4 LR: 0x9E03

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/3269124/vita3k.log)

**Comment by rfghgxfc** [comment]
Hello Man! Recently I was looking for similar material but for the i7 8700K + GTX 1080 TI + 16 GB RAM DDR 4 2133 kernel. When I launch the To Love-Ru Trouble Darkness: True Princess [PCSG00689] game, I have an error on the emulator and the cmd command gives the following: [19 : 45: 16.551] | E | [logging :: ret_error_impl]: sceKernelAllocMemBlock returned SCE_KERNEL_ERROR_NO_MEMORY (0x8002710B)
[19: 45: 16.552] | C | [run]: Unicorn error 0x7 at: start PC: 0x82E023D0 error PC 0x83F83FD4 LR: 0x9E03. How to solve this problem !? I run on the Vita3K emulator

**Comment by Minihornet** [comment]
Bit late to the party but for me it shows menu but there are missing file errors: log vv
[vita3k.log](https://github.com/Vita3K/compatibility/files/4593536/vita3k.log)

**Comment by DracoSafarius** [comment]
Yeah I get the same popup error as Minihornet, file it tries to pull doesn't even exist, whole folder doesn't

**Comment by Zangetsu38** [comment]
is unity game, for now is not supported

**Comment by kaishi888** [comment]
[vita3k.log](https://github.com/Vita3K/compatibility/files/6642527/vita3k.log)
<img width="721" alt="屏幕截图 2021-06-12 234230" src="https://user-images.githubusercontent.com/79926873/121782618-fcd51a00-cbdc-11eb-9a37-fc6664377de6.png">
it stuck in loading window

**Comment by THEYOUTOBER1234** [comment]
how do i get the zip/vpk file?

**Comment by DracoSafarius** [comment]
Not sure if we can provide ways to find those here, so I won't get fully into it other than say just google the ID code

**Comment by THEYOUTOBER1234** [comment]
> Not sure if we can provide ways to find those here, so I won't get fully into it other than say just google the ID code

Fair

**Comment by othmanead** [comment]
# Vita3K summary
- Version: v0.1.5
- Build number: 2627
- Commit hash: https://github.com/vita3k/vita3k/commit/a31fd85c
- CPU backend: Dynamic  

# Test environment summary
- Tested by: othmanead
- OS: Windows 11
- CPU: Intel i5-10300H
- GPU: RTX 3060 mobile
- RAM: 16 GB


# Issue: 
- Games plays perfectly

# Screenshots:
![image](https://user-images.githubusercontent.com/106798110/172032365-21d7b0d3-0555-46f0-9edf-6b9286ba396a.png)

![image](https://user-images.githubusercontent.com/106798110/172032398-e3664c18-1328-4995-abce-bc2b1d15287b.png)

![image](https://user-images.githubusercontent.com/106798110/172032452-717abd16-5f34-4fb7-a1e4-1db66c4ce166.png)



# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/8839219/vita3k.log)





# Recommended labels

- Playable

**Comment by rfghgxfc** [comment]
How man, How?

**Comment by rfghgxfc** [comment]
![1](https://user-images.githubusercontent.com/53576226/172190275-b1754ed9-8d4c-4ce4-9051-20ea1c159a26.jpg)

Success

**Comment by chadness-l** [comment]
![image](https://github.com/Vita3K/compatibility/assets/144375065/0bbf2fa3-96d6-44e6-bc83-e8a115ab97e5)
what is that pls help

**Comment by rfghgxfc** [comment]
> ![image](https://user-images.githubusercontent.com/144375065/266394183-0bbf2fa3-96d6-44e6-bc83-e8a115ab97e5.png) what is that pls help

Maybe Re-install Emulator Vita3K. (Google in help)

**Comment by tyarumasa** [comment]
I tested this again on the latest Windows build.

Version:
- Vita3K v0.2.1-4067-be692710

The issue is still reproducible.

Additional findings:

- Opening the pause menu → Data → closing it → immediately opening Data again causes a softlock with a high reproduction rate.
- During this softlock, the BGM continues playing and the FPS counter continues updating, but no input is accepted.
- The game can also softlock during scene transitions (screen fades). In this case, the BGM continues playing but the FPS counter freezes.
- The issue still occurs with:
  - Vulkan
  - OpenGL
  - Internal Resolution 1x
  - Older Vita3K builds

This seems to be related to menu/scene transitions rather than graphics settings.
